docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -58,6 +58,47 @@ PostgreSQL-specific connector:
 </dependency>
 ----
 
+Since all those artifacts are in Hazelcast Private Repository, you also have to add:
```

Comment:
'Since' can confuse some non-native speakers so suggesting:
```suggestion
Because all these artifacts are in Hazelcast's private repository, you also have to add:
```